src/common/models/Projection.ts

```diff
@@ -485,19 +484,31 @@ export class Projection {
         dates: [],
       };
     }
-    let earliestDate, latestDate;
+    let earliestDate, latestDate, year, month, date;
     // If we have projections, we use that time range; else we use the actuals.
     // TODO(chris): Is there a reason that this was bound to the projections timeseries first?
     // It cuts off some of the earlier dates
     if (metricsTimeseriesRaw.length > 0) {
-      earliestDate = moment.utc(first(metricsTimeseriesRaw)!.date);
-      latestDate = moment.utc(last(metricsTimeseriesRaw)!.date);
+      [year, month, date] = first(metricsTimeseriesRaw)!
+        .date.split('-')
+        .map(elem => parseInt(elem));
+      earliestDate = new Date(Date.UTC(year, month - 1, date));
```

Comment:
`Date.UTC` accepts integers as parameters, not strings. Hence the string manipulation for `[year, month, date]`.

Reply:
Hrm, I think you can just do `new Date(first(metricsTimeseriesRaw)!.date)` since these give me the same result:
![image](https://user-images.githubusercontent.com/206364/115610147-5126e200-a29d-11eb-939b-d4f37d2fa7ee.png)

or am I missing something?

Reply:
I decided to use `Date.UTC` to be explicit about instantiating it as a UTC timestamp and to be extra cautious due to the note outlined under "3. Timestamp string" in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date. That said, switching it can potentially make it much more straightforward, and you are correct that parsing _just_ the date value as string (no timestamp attached) should yield the exact same result (but only if we always parse date-only value and in ISO8601 format). I can make the switch to make it more straightforward.